addons/website/tests/test_website_visitor.py

```diff
@@ -380,10 +377,10 @@ def _prepare_main_visitor_data(self):
 
     def _prepare_linked_visitor_data(self):
         return {
-            'name': "Mitchell Main's Phone",
             'lang_id': self.env.ref('base.lang_en').id,
             'country_id': self.env.ref('base.be').id,
             'website_id': 1,
+            'access_token': 'f9d2ed295274b7814778f76a88c72712dcf8d5f5',
```

Comment:
As this tool is used in several tests, I think it is a bad idea to hardcode a token. You could get the same hash token and therefore the same identity without knowing it. You should randomize it at each call to this method to ensure a new token is generated.